examples/auto_totem.js

```diff
@@ -0,0 +1,24 @@
+/*
+ * This script will automaticly set if a totem is in the inventory the totem in the off-hand.
+ * It checks for a totem every secound.
+ */
+const mineflayer = require('mineflayer')
+
+if (process.argv.length < 4 || process.argv.length > 6) {
+  console.log('Usage : node echo.js <host> <port> [<name>] [<password>]')
+  process.exit(1)
+}
+
+const bot = mineflayer.createBot({
+  host: process.argv[2],
+  port: parseInt(process.argv[3]),
+  username: process.argv[4] ? process.argv[4] : 'totem',
+  password: process.argv[5]
+})
+
+setInterval(() => {
+  var totem = bot.inventory.findInventoryItem(449, null)
+  if (totem && totem.type === 449) {
```

Comment:
no need to check again the type of the object

Reply:
@Karang The player might have no totem in his inventory. That would return null. If i try to read the property type of a undefined object if would throw an error.